tslint/tslint_report.js

```diff
@@ -3,6 +3,27 @@
 
 const { getAnnotation } = require('./tslint_annotations')
 const { countIssueLevels } = require('../annotations_levels')
+const { config } = require('../config')
+
+/**
+ * @param {*} issues the issues found by TSLint
+ */
+function createMoreInfoLinks (issues) {
+  let issuesMap = new Map()
+  let moreInfo = ''
+
+  for (let issue of issues) {
+    if (issuesMap.has(issue.ruleName) === false) {
+      issuesMap.set(issue.ruleName)
+      const linkToDoc = config.tslintProjectWebsite + '#' + issue.ruleName
+      moreInfo += `[${issue.ruleName}](${linkToDoc})\n`
+    }
+  }
+  if (issuesMap.size <= 0) {
```

Comment:
We don't need this check because if issues array is empty then it won't go into to for loop and moreInfo will be an empty string.